lib/zclient.h

```diff
@@ -328,6 +329,7 @@ struct zapi_route {
 	 * This is an enum but we are going to treat it as a uint8_t
 	 * for purpose of encoding/decoding
 	 */
+	afi_t  afi;
```

Comment:
Please don't align variable declarations like this.

Reply:
The AFI value is embedded in the `prefix` member a few lines below. I don't see the benefit of replicating this information here.

Now we have places consulting `api.afi` and others consulting `api.prefix.family` (which can be converted to an AFI value using `family2afi()`). I think this opens room for inconsistencies and confusion.

Reply:
code changes done